javascript/src/input_selectize.js

```diff
@@ -46,7 +47,9 @@ input.register({
         lastKnownKeys = keyArray;
         ctHandle.set(keyArray);
       }
-    });
+    }
+    selectize.on("change", updateFilter);
+    updateFilter();
```

Comment:
This call is too early for `runtime: shiny`.. I think we should use something like `HTMLWidgets.addPostRenderHandler()` to schedule this to run later?